test/integration_test.py

```diff
@@ -747,6 +745,8 @@ def _test_repository_files(self, catalog: str):
             else:
                 while response['Status'] != 302:
                     self.assertEqual(301, response['Status'])
+                    response = self._get_url_json(furl(response['Location']))
+
                     response = self._get_url_json(response['Location'])
 
                 response = self._get_url(response['Location'], stream=True)
```

Comment:
This is broken. It could have been caught by running the IT or paying attention to PyCharm's warnings.